ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
 - agent: kubernetes
-  cluster: api.ci
+  cluster: build02
```

Comment:
/hold

This one uses GH credentials that are not present on build farms. Either this needs to run on app.ci or this could use some special credentials synced normally via mirroring.

Reply:
Updated.